### ISSUE TITLE
field types

### ISSUE THREAD

**Issue by perler** [makeITyourway/graylog2_nginx_acecss.log_json#1]
nice work, but the problem is, that all field types are set to string inside graylog, even the numeric ones like response time etc. which prevents making response time graphs like these: http://docs.graylog.org/en/2.2/_images/dashboards_5.png

**Comment by mschiessl** [comment]
ah i see the issue - thank you for the hint - i'll try to fix them ;)

**Comment by mschiessl** [comment]
ok, after i had a look at this, i guess the issue is the regex in the end where it pareses everything as string. still have not a real idea on how to solve this - but i'll stick to it and hopefully come back with a working solution.

**Comment by petestorey26** [comment]
The solution is not to put "" around the numeric types in the nginx json log_format - if you remove them, then it understands it and it's fine.  From a JSON point of view, putting "" round something makes it a string.

**Comment by mschiessl** [comment]
yeah - that it is @petestorey26  ! i'll gonna fix that one out - 
thanks for sorting this out!

**Comment by mschiessl** [comment]
should be fixed here:
https://github.com/makeITyourway/graylog2_nginx_acecss.log_json/commit/2169928e357dd844ae6ae2ca1ccb4d3565924442
